SPECS/perl-App-cpanminus/perl-App-cpanminus.spec

```diff
@@ -0,0 +1,513 @@
+# to check for correct unpacking. So we need non-optional run-time
+# dependencies at build-time too:
+BuildRequires:  perl(Config)
+BuildRequires:  perl(constant)
+# CPAN::DistnameInfo not needed for compilation
+# CPAN::Meta not needed for copmilation
+# CPAN::Meta::Check not needed for compilation
+BuildRequires:  perl(CPAN::Meta::Requirements)
+# CPAN::Meta::YAML not needed for compilation
+BuildRequires:  perl(Cwd)
+# Digest::SHA not needed for compilation
+# ExtUtils::Manifest not needed for compilation
+BuildRequires:  perl(File::Basename)
+BuildRequires:  perl(File::Copy)
+BuildRequires:  perl(File::Find)
+# File::HomeDir not needed for compilation
+# File::pushd not needed for compilation
+BuildRequires:  perl(File::Temp)
+# HTTP::Tiny not needed for compilation
+# JSON::PP not needed for compilation
+# local::lib not needed for compilation
+# LWP::Protocol::https not needed for compilation
+# LWP::UserAgent not needed for compilation
+# Module::CoreList not needed for compilation
+# Module::CPANfile not needed for compilation
+# Module::Metadata not needed for compilation
+# Module::Signature not needed for compilation
+# Parse::PMFile not needed for compilation
+# Safe not needed for compilation
+BuildRequires:  perl-String-ShellQuote
+BuildRequires:  perl(Symbol)
+BuildRequires:  perl(version)
+# version::vpp not needed
+# Win32 not used
+# YAML not needed for compilation
+# Tests:
+BuildRequires:  perl(Test::More)
+# There is no way to install core non-duallived modules from CPAN,
+# (e.g. Path-Tiny CPAN distribution requiers "open" module), require full Perl
+# for that.
+Requires:       perl
+# Current dependency generator cannot parse compressed code. Use PPI to find
+# them, and list them manually:
+# Archive::Tar is optional
+# Archive::Zip is optional
+# Compress::Zlib is optional
+Requires:       perl-CPAN-DistnameInfo
+Requires:       perl(CPAN::Meta)
+Requires:       perl-CPAN-Meta-Check
+Requires:       perl(CPAN::Meta::YAML)
+Requires:       perl(Digest::SHA)
+Requires:       perl(ExtUtils::Install) >= 1.46
+Requires:       perl(ExtUtils::MakeMaker) >= 6.58
+Requires:       perl(ExtUtils::Manifest)
+# File::HomeDir is optional
+Requires:       perl-File-pushd
+# HTTP getter by LWP::UserAgent or wget or curl or HTTP::Tiny
+Requires:       perl(HTTP::Tiny)
+Requires:       perl-local-lib
+# LWP::Protocol::https is optional
+# LWP::UserAgent is optional
+Requires:       perl-Module-Build
+Requires:       perl(Module::CoreList)
+Requires:       perl-Module-CPANfile
+Requires:       perl(Module::Metadata)
+# Module::Signature is optional
+Requires:       perl-Parse-PMFile
+Requires:       perl(Safe)
+# version::vpp not used
+# Win32 not used
+Requires:       perl-YAML
+# XXX: Keep Provides: cpanminus to allow `yum install cpanminus' instead of
+# longer `yum install perl-App-cpanminus'.
+Provides:       cpanminus = %{version}-%{release}
+Obsoletes:      cpanminus <= 1.2002
```

Comment:
This "Obsoletes" line is not necessary here, as we've never shipped a "cpanminus" package.

Reply:
Removed.